backend/migrations/dataall_migrations/versions/initial.py

```diff
@@ -0,0 +1,25 @@
+from migrations.dataall_migrations.base_migration import MigrationBase
+from migrations.dataall_migrations.versions.remove_wildcard_share_policy import RemoveWildCard
+from typing import Type
+
+
+class InitMigration(MigrationBase):
```

Comment:
I am thinking it over again and I don't think the initial migration (dummy node) is required. We can just do `mm = MigrationManager(RemoveWildcard, current_key)`